OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -72,7 +72,10 @@ public class MobileInfo : PausableConditionalTraitInfo, IMoveInfo, IPositionable
 		public readonly bool CanMoveBackward = false;
 
 		[Desc("After how many ticks the actor will turn forward during backoff")]
-		public readonly int BackwardDuration = 40;
+		public readonly int BackwardDuration = int.MaxValue;
```

Comment:
I'd say leave this value as is and remove the update rule

Reply:
Ummm, Idk, but at the time Milk and me were having discussion Milk told me in his opinion,  the most of the `Backward` movement usecase should be real life vehicle units (consider cell only), and then, the hover unit (like our style of setting which consider the time).

So I change the default setting based on this idea. If people want to make a RA1 style tank can move backward in short distance (which makes sence, real life tanks like that), they don't have to change the backward time as well.

Reply:
This isn't IRL though, the distance to enemy is much, much smaller and there's much less space to manoeuvre. So in an active combat engagement backwards drive makes a lot of sense. For all vehicles. The goal is to quickly reposition to avoid enemy fire.

Reply:
have drop that commit.